src/redis-cli.c

```diff
@@ -763,7 +763,7 @@ static void removeUnsupportedArgs(struct cliCommandArg *args, int *numargs, sds
             i++;
             continue;
         }
-        for (j = i; j != *numargs; j++) {
+        for (j = i; j != *numargs - 1; j++) {
```

Comment:
this was an invalid read that happens whenever we get here, which means that this case isn't covered by the tests?

what if `i` == `*numargs`? we'll get an infinite loop?
maybe we should change the exit condition to `<` instead of adding the `-1`?

Reply:
`i` can't be `*numargs` because of the outer loop condition.
this is a zero based index... `<` is not good as we will enter the loop for the last element (or maybe I missed your point)